src/actinia_statistic_plugin/vector_sampling.py

```diff
@@ -0,0 +1,233 @@
+# -*- coding: utf-8 -*-
+"""
+Perform vector map sampling on a vector map layer based on input points.
+"""
+
+import pickle
+import tempfile
+import json
+from flask import jsonify, make_response
+from copy import deepcopy
+from flask_restful_swagger_2 import swagger
+from flask_restful_swagger_2 import Schema
+from actinia_core.models.response_models import ProcessingErrorResponseModel
+from actinia_core.rest.ephemeral_processing import EphemeralProcessing
+from actinia_core.rest.resource_base import ResourceBase
+from actinia_core.core.common.redis_interface import enqueue_job
+from flask.json import dumps
+from actinia_core.core.common.app import auth
+from actinia_core.core.common.api_logger import log_api_call
+from .response_models import VectorSamplingResponseModel
+
+
+__license__ = "GPLv3"
+__author__ = "Sören Gebbert, Markus Neteler"
+__copyright__ = "Copyright 2016-present, Sören Gebbert and mundialis GmbH & Co. KG"
+
+
+class PointListModel(Schema):
+    """This schema defines the JSON input of the vector sampling resource"""
+
+    type = "object"
+    properties = {
+        "points": {
+            "type": "array",
+            "items": {
+                "type": "array",
+                "items": {"type": "string", "maxItems": 3, "minItems": 3},
+            },
+            "description": "A list of coordinate points with unique ids [(id, x, y), (id, x, y), (id, x, y)]",
+        }
+    }
+    example = {"points": [["a", "1", "1"], ["b", "2", "2"], ["c", "3", "3"]]}
```

Comment:
Also out of curiosity, why do we need the `id`? Is this the same elsewhere? Because we do not actually make use of the id, do we?

Reply:
Well, I followed the STRDS code here:

https://github.com/mundialis/actinia_statistic_plugin/blob/e434056c9ec9ad50f28f5ecc6a375f5d8d5aee14/src/actinia_statistic_plugin/strds_sampling.py#L177

Reply:
The `id` is used in the response. So it is maybe helpful that the user has not to check the coordinates to know which point is meant.